docs/runbooks/NoLeadingVirtOperator.md

```diff
@@ -1,42 +1,73 @@
+<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
+
 # NoLeadingVirtOperator 
 
 ## Meaning
 
-This alert is triggered when the system detects that in the past 10 minutes, there is no virt-operator pod holding the leader lease despite there are one or more virt-operator pod is in `Ready` state. The alert suggests that there is no operating virt-operator pod. 
+This alert fires when the system detects that in the past 10 minutes, no `virt-operator` pod has been holding the leader lease, although one or more virt-operator pods are in the `Ready` state. The alert suggests that there is no operating `virt-operator` pod. 
 
-## Impact
+The `virt-operator` is the first operator to start in a cluster. Its primary responsibilities include the following: 
+
+- Installing, live updating, and live upgrading a cluster
+
+- Monitoring the life cycle of top-level controllers, such as `virt-controller`, `virt-handler`, `virt-launcher`, and managing their reconciliation
+
+- Certain cluster-wide tasks, such as certificate rotation and infrastructure management
 
-The virt-operator is the first k8s operator coming alive in a KubeVirt cluster. Its primary responsibilities are installation, live-update, live-upgrade of a KubeVirt cluster, monitoring the life-cycle of top-level controllers, such as virt-controller, virt-handler, virt-launcher, etc. and manage their reconciliation. In addition, virt-operator is responsible for cluster-wide tasks, such as certificate rotation and some infrastructure management, etc.  The virt-operator deployment has a default replica of 2 pods, with one leader pod holding a leader lease, indicating an operating virt-operator pod. 
+The `virt-operator` deployment has a default replica of 2 pods, with one leader pod holding a leader lease, indicating an operating `virt-operator` pod. 
```

Comment:
with one leader pod holding a leader lease > with one pod holding a leader lease

You can remove ", indicating an operating `virt-operator` pod" because it is redundant. A leader pod is the one holding the leader lease. "operating" does not fit this context.